test/format/ops.c

```diff
@@ -158,11 +158,15 @@ operations(u_int ops_seconds, bool lastrun)
     if (!SINGLETHREADED)
         g.rand_log_stop = true;
 
-    /* Logging requires a session. */
-    if (g.logging)
-        testutil_check(conn->open_session(conn, NULL, NULL, &session));
+    /* Logging and locking require a session. */
+    testutil_check(conn->open_session(conn, NULL, NULL, &session));
```

Comment:
This comment was already wrong (the session is used by the RNG a few lines further down), I'd just toss it, we just "need a session".

Reply:
Done